test/grpc/route_guide/route-chat.cc

```diff
@@ -0,0 +1,80 @@
+#include <iostream>
+#include <string>
+
+#include "eventuals/do-all.h"
+#include "eventuals/event-loop.h"
+#include "eventuals/finally.h"
+#include "eventuals/foreach.h"
+#include "eventuals/head.h"
+#include "eventuals/let.h"
+#include "eventuals/promisify.h"
+#include "eventuals/then.h"
+#include "gtest/gtest.h"
+#include "test/grpc/route_guide/route-guide-test.h"
+
+using grpc::Status;
+
+using eventuals::DoAll;
+using eventuals::Finally;
+using eventuals::Iterate;
+using eventuals::operator*;
+using eventuals::Foreach;
+using eventuals::Head;
+using eventuals::Let;
+using eventuals::Then;
+using eventuals::grpc::Stream;
```

Comment:
Please go through every single file and confirm that you are only bringing in things that you use.